### PR TITLE
#166789950 Run Codeclimate Test Coverage

### PR DESCRIPTION
**Pull request by niyoceles**
#### What does this PR do?
- Configure codeclimate test coverage
#### Description of Task to be completed
- Enables run code climate test coverage 
#### How should this be manually tested?
```bash
git clone https://github.com/andela/ah-92explorers-frontend.git
cd ah-92explorers-frontend
git checkout ch-run-codeclimate-test-cov-166789950
```
- Open README file to see the badges
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166789950](https://www.pivotaltracker.com/story/show/166789950)
#### Screenshots (if appropriate)
Screen Shot 2019-08-16 at 16 08 01

#### Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

**Review by IsaiahRn — APPROVED** [review]

**Review by Hervera — APPROVED** [review]
Ready to be merged

File: package.json [hunk 1]
```diff
@@ -8,8 +8,7 @@
     "test": "jest",
     "test:cov": "jest --coverage",
     "test:watch": "jest --watchAll",
-    "cypress": "cypress run",
-    "cy:open":"cypress open",
+    "cypress": "cypress open",
```
Comment (kipropbrian) [inline package.json]: Check up with @IsaiahRn on this. Why did you delete `cypress run`. It should be left for it to run on CLI.

File: jest.config.js [hunk 2]
```diff
@@ -5,4 +5,13 @@ module.exports = {
   testPathIgnorePatterns: [
     './src/__tests__/setup/',
   ],
+  "collectCoverageFrom": [
```
Comment (kipropbrian) [inline jest.config.js:8]: Should also ignore cypress folder

**Comment by kipropbrian** [comment]
Could you add a screenshot of the coverage report. Also Codeclimate should be posting the report here. Right? I cannot see it.

File: package.json (hunk 1 above)
Comment (niyoceles) [reply package.json]: Solved.

File: jest.config.js (hunk 2 above)
Comment (niyoceles) [reply jest.config.js:8]: Solved.
